app/assets/javascripts/articles.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
When using generators to create new files, it's common to delete the empty ones if they're not going to be used in the near future. This is a style choice each project can do, and some choose to leave them be as well.

For this project I don't think it matters that much, but for larger projects with hundreds or thousands of files, it matters more.